sdk/src/contract-types/CraftingFacet.ts

```diff
@@ -274,8 +274,8 @@ export interface CraftingFacet extends BaseContract {
   };
 
   addRecipe(
-    recipe: RecipeStruct,
-    overrides?: Overrides & { from?: PromiseOrValue<string> }
+    recipe: RecipeStruct
+    // overrides?: Overrides & { from?: PromiseOrValue<string> }
```

Comment:
Same thing, why remove this line of code?

Reply:
And if it doesn't belong, let us remove it instead of commenting it out.

Reply:
Removing!